### PR TITLE
Allow arbitrary directory for ern root

### PR DESCRIPTION
**Pull request by belemaire**
Through `ERN_HOME` environment variable.

This is useful in a few contexts but mostly for CI env, as it allows to keep the whole `.ern` directory under the job workspace.

### CODE REVIEW

File: global-cli/src/index.js [hunk 1]
```diff
@@ -82,58 +89,93 @@ function firstTimeInstall () {
...
+    installProc.on('close', function(code) {
       if (code === 0) {
-        spinner.succeed(`Hurray ! Electrode Native v${initialVersion} was successfully installed.`)
+        spinner.succeed(
+          `Hurray ! Electrode Native v${initialVersion} was successfully installed.`
+        )
       } else {
-        spinner.fail(`Something went wrong :( Run the command again with --debug flag for more info.`)
+        spinner.fail(
+          `Something went wrong :( Run the command again with --debug flag for more info.`
```
Comment (krunalsshah) [inline global-cli/src/index.js]: not part of the PR though please change to `'` , multiple places.

File: global-cli/yarn.lock [hunk 2]
```diff
@@ -112,9 +136,9 @@ error-ex@^1.2.0:
   dependencies:
     is-arrayish "^0.2.1"
 
-escape-string-regexp@^1.0.2:
+escape-string-regexp@^1.0.2, escape-string-regexp@^1.0.5:
   version "1.0.5"
-  resolved "https://registry.yarnpkg.com/escape-string-regexp/-/escape-string-regexp-1.0.5.tgz#1b61c0562190a8dff6ae3bb2cf0200ca130b86d4"
+  resolved "http://npme.walmart.com/escape-string-regexp/-/escape-string-regexp-1.0.5.tgz#1b61c0562190a8dff6ae3bb2cf0200ca130b86d4"
```
Comment (krunalsshah) [inline global-cli/yarn.lock]: oops :(

File: global-cli/yarn.lock [hunk 3]
```diff
@@ -16,6 +16,12 @@ ansi-styles@^2.2.1:
   version "2.2.1"
   resolved "https://registry.yarnpkg.com/ansi-styles/-/ansi-styles-2.2.1.tgz#b432dd3358b634cf75e1e4664368240533c1ddbe"
 
+ansi-styles@^3.2.1:
+  version "3.2.1"
+  resolved "http://npme.walmart.com/ansi-styles/-/ansi-styles-3.2.1.tgz#41fbb20243e50b12be0f04b8dedbf07520ce841d"
```
Comment (krunalsshah) [inline global-cli/yarn.lock]: change it to `registry.yarnpkg.com` , multiple places.

**Comment by belemaire** [comment]
@krunalsshah Fixed the `yarn.lock` issues. 
For the rest (backtick) let's do it at another time, it does not really matter that much.

**Comment by krunalsshah** [comment]
@belemaire looks now all dependency are pointing to `walmart` npme

**Comment by belemaire** [comment]
@krunalsshah Damn :( Should be all good now
